gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStream.java

```diff
@@ -145,7 +145,11 @@ public void readFully(long position, byte[] buffer, int offset, int length) thro
           nread += nbytes;
         }
       } finally {
-        seek(oldPos);
+        try {
+          seek(oldPos);
+        } catch (IOException ie) {
+          logger.atFiner().log("Ignoring IOE on seek to (%d): (%s)", oldPos, ie.getMessage());
```

Comment:
I think this should be an error

Reply:
Also now the Stream might be in an inconsistent state. I am not sure ignoring this is the right thing to do.

Reply:
Not really - if you look into the implementation of seek - it will only throw error if your position is not correct. That will never happen in this scenario as we are setting it to previous old position only. That's why this exception should never occcur.